SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialActorUtils.h

```diff
@@ -85,13 +86,9 @@ inline FVector GetActorSpatialPosition(const AActor* InActor)
 	}
 	else if (const APlayerController* PlayerController = Cast<APlayerController>(Controller))
 	{
-		if (PlayerController->IsInState(NAME_Spectating))
+		if (USceneComponent* RootComponent = PlayerController->GetRootComponent())
```

Comment:
@sjtsilvester this seems like a pretty big change, context would be good

Reply:
This was a change Intrepid made that we didn't catch at the time. The CL for this has the comment 

"Fixing snap to ground on server and remotes.
Tiny fix for SiegeTeleport to forcenetupdate when assigned siege team type"

but still unsure - they had an issue where players would teleport back to their original location when being teleported to the siege island so it may have been an attempt to fix that. I will ask.

Reply:
Maybe this is a revert then?

Reply:
Have asked Intrepid

Reply:
Yep, this is a revert